FoodOrderApi/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using AutoMapper;
+using FoodOrderApi.CustomActionFilter;
+using FoodOrderApi.DataProvider;
+using FoodOrderApi.Model.Domain;
+using FoodOrderApi.Model.DTO;
+using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+
+namespace FoodOrderApi.Controllers
+{
+    [Route("Customer")]
+    public class CustomerController : ControllerBase
+    {
+        private IDataProvider _dataProvider;
+        private readonly IMapper mapper;
+        private readonly ILogger logger;
+
+        public CustomerController(IDataProvider dataProvider, IMapper mapper, ILogger<CustomerController> logger)
+        {
+            _dataProvider = dataProvider;
+            this.mapper = mapper;
+            this.logger = logger;
+        }
+
+        [HttpPost("PlaceOrder")]
+        [ValidateModel]
```

Comment:
Why not use this attribute for other actions also?

Reply:
Here, we are receiving it as list of object, So, we have used ValidateModel attribute.
For other actions, either Guid or string is validated while receiving (using [Required]).